src/py21cmmc/likelihood.py

```diff
@@ -1782,3 +1882,136 @@ def computeLikelihood(self, model):
             )
             return -np.inf
         return lnl
+
+class Likelihood1DPowerLightconeUpper(Likelihood1DPowerLightcone):
+    r"""
+    Likelihood based on Chi^2 comparison to luminosity function data.
+
+    Parameters
+    ----------
+    datafile : str, optional
+        Input data should be in a `.npz` file, and contain the arrays:
+        * ``z_bands``: the redshift of the observation bands
+        * ``bandx``: the power spectrum upper limits at redshift x
+        * ``wfbandx``: the window function for band x
+        * ``kwfbandx``: the k values [/Mpc] of the window function for band x
+    """
+
+    def __init__(self, datafile="", data = None, name="", 
+                 redshifts = np.array([10.37213048, 7.92876298]),
```

Comment:
same as redshift, you cant really change the redshift to other values, in particular when they are far away from 8 and 10